### DISCUSSION TITLE
Contributions not appearing on account

### DISCUSSION THREAD

**Discussion by Austinae** [community/community#23068] [New to GitHub]
<p>Hello there,<br>
I’ve noticed that I can’t see the contributions I’ve made on my account. I’ve pushed a few times today on my repos’ default branches but it’s not working, example with my YoutubeMSCDWNLDER repo that I updated 5 min ago.</p>
<p>I know this much but it’s not helping:</p>
<p>Commits will appear on your contributions graph if they meet all of the following conditions:</p>
<p>The email address used for the commits is associated with your GitHub account.<br>
The commits were made in a standalone repository, not a fork.<br>
The commits were made:<br>
In the repository’s default branch (usually master)<br>
In the gh-pages branch (for repositories with project sites)<br>
I’ve also read this to make sure I’m not mad:<br>
<a href="https://help.github.com/en/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites" class="onebox" target="_blank" rel="nofollow noopener">https://help.github.com/en/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites</a></p>
<p>Any help is appreciated.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>As the repo you mention is private, I can’t check for sure, but usually when commits don’t show up it’s because the author email address in the Git metadata doesn’t match an email address linked to your account.</p>
<p>Here’s how you can check what author email is being used by adding <code>.patch</code> to the URL:</p>
<p><a href="https://help.github.com/en/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile#you-havent-added-your-local-git-commit-email-to-your-profile" class="onebox" target="_blank">https://help.github.com/en/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile#you-havent-added-your-local-git-commit-email-to-your-profile</a></p>
<p>If it’s an email address that isn’t currently linked to your account, you can add it to your account to claim those contributions. The email address doesn’t have to work or be verified to claim contributions.</p>
<p>If it’s a non-email address string, then you’ll need to rewrite the repo’s history:</p>
<p><a href="https://help.github.com/en/github/using-git/changing-author-info" class="onebox" target="_blank">https://help.github.com/en/github/using-git/changing-author-info</a></p>
<p>After making any sort of change, please allow 24 hours for the contribution graph to update!</p>

**Reply by sitoftonic** [reply]
@yamiacat Thanks for the information. I'm currently trying to understand what's happening to my profile, because since I anonymized my email and set up Github's auto-generated email for my commits I haven't seen any contributions in my profile. I have tried to add that email to my profile but it shows an error saying "The email already exists", which is obvious because it's my own anonymized email here in Github. Do you know what should I do to bring back my contributions while mantaining my email private?

**Reply by yamiacat** [reply]
Hi sitoftonic! At a guess there's a typo in the email address as it's entered into Git's `user.email` field - that's the only reason I can think of for commits authored with your `noreply` address from not showing up!

Do you have a link to a specific commit that should be showing up as a contribution but isn't?

**Reply by sitoftonic** [reply]
Thank you for your fast reply @yamiacat! It looks like the email is setup correctly on my end. I have generated a dummy commit into one of my public repo's here for you to check it out: https://github.com/sitoftonic/email-checker/commit/95b20756a29250c7021700ac38db432c380b3b81

**Edit**: It looks like contributions are showing now for the commits I did in the attached public repo! So is there a problem with my private repositories? I'm using the exact same git config for them.

**Reply by airtower-luna** [reply]
You can configure whether the contributions section shows only contributions in public repositories, or private ones, too. Click on "Contribution settings" to the top right of the green square graph.

**Reply by sitoftonic** [reply]
@airtower-luna The strange thing is that I already have the check, so it should be showing the private contributions too, right?
![image](https://user-images.githubusercontent.com/26420881/191574081-efd01bd5-9ba3-4538-93cc-3721feec03ad.png)

**Reply by airtower-luna** [reply]
They should, yes, under the [same rules as for public contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commits) otherwise.

**Reply by sitoftonic** [reply]
@airtower-luna Thank you so much for the link and your time... Have checked it before but after re-reading it I now realized the issue. It happens that commits I have been doing these days are not being pushed into the default branches of respective repo's. It's true then that when those commits merge with the default branch contributions will be counted.

**Comment by danielChuks** [comment]
<p>Also i read all the documentation on that regards and i can see i have not gone against anything stated there i wonder why they are not visible</p>

**Comment by danielChuks** [comment]
<p>Hello Team,</p>
<p>I have been making a lot of commit but this contributions don’t appear on my profile.</p>
<p>Kindly help me out with this thanks.</p>

**Comment by jfullstackdev** [comment]
<p>just a quick check,</p>
<p>your commits are not verified, you want to make sure it’s verified, I don’t know what is your setup for that</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/0/40dcc72b16342e62124c11a225aaf38bf07e2b68.png" data-download-href="/uploads/short-url/9fNzvP2jSVK4J8kwQB5FaqT4Y5O.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074671-e4252152-e166-419f-9f91-cd7fe943f77d.png" alt="image" data-base62-sha1="9fNzvP2jSVK4J8kwQB5FaqT4Y5O" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40dcc72b16342e62124c11a225aaf38bf07e2b68_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40dcc72b16342e62124c11a225aaf38bf07e2b68_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40dcc72b16342e62124c11a225aaf38bf07e2b68_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40dcc72b16342e62124c11a225aaf38bf07e2b68_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 57.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by danielChuks** [comment]
<p>Hell Team,</p>
<p>Thank your for your email.</p>
<p>I have not been doing anything different from what I’m doing.</p>
<p>How do I get my commits verified??</p>
<p>Because I using my PC and everything is as it use to be.</p>
<p>Just contributions not coming up.</p>

**Comment by danielChuks** [comment]
Hi

I face same issue but this got resolved when I use git desktop.

Although I’m a fan using the terminal but that was not reflecting my
contributions so I had resort to git desktop.

Hope this helps.

**Comment by jfullstackdev** [comment]
I'm using GitHub Desktop without Git, the trick if you can't find any other solution is create a branch and make a pull request on GitHub, this way, the verification is set because you did that on GitHub itself
